plugins/CircuitViewer/io/utils.h

```diff
@@ -27,8 +27,8 @@ namespace brayns
 {
 namespace
 {
-brion::GIDSet _gidsFromRange(const uint32_t first, const uint32_t last,
-                             const double fraction, const int32_t seed)
+inline brion::GIDSet _gidsFromRange(const uint32_t first, const uint32_t last,
```

Comment:
I somewhat had to make this inline in order to avoid a 'defined but not used [-Werror=unused-function]' compilation error :-/